assets/component-cart-items.css

```diff
@@ -224,6 +236,10 @@ cart-remove-button .icon-remove {
     border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.2);
   }
 
+  .cart-items thead th:first-child {
+    width: 66%;
```

Comment:
On large screen, I reduce this width to 60% to feel comfortable.
As soon as a reduce the device width, I feel like I would like to adjust the width again, do you think we can add a breakpoint?

[Video recording](https://screenshot.click/21-08-2gdxm-rbp6i.mp4)